test/WebSites/Identity.DefaultUI.WebSite/StartupBase.cs

```diff
@@ -44,7 +45,12 @@ public virtual void ConfigureServices(IServiceCollection services)
                 .AddRoles<IdentityRole>()
                 .AddEntityFrameworkStores<TContext>();
 
-            services.AddMvc();
+            services.AddMvc()
+                .AddRazorOptions(ro =>
+                {
+                    ro.FileProviders.Clear();
```

Comment:
Maybe add a comment that says why this is being done.

Reply:
Sure